presto-orc/src/test/java/com/facebook/presto/orc/TestOrcWriter.java

```diff
@@ -120,4 +125,86 @@ public void testWriteOutputStreamsInOrder()
             }
         }
     }
+
+    @Test(expectedExceptions = IllegalStateException.class)
+    public void testVerifyIllegalStateException()
+            throws IOException
+    {
+        for (OrcWriteValidationMode validationMode : OrcWriteValidationMode.values()) {
```

Comment:
Remove validation given that's not the purpose of the test. Just use `false` and `HASH` for validation options.